configuration/scripts/options/set_nml.snwgrain

```diff
@@ -1,3 +1,13 @@
-tr_snow = .true.
-snwgrain = .true.
+tr_snow         = .true.
+snwredist       = 'ITDrdg'
```

Comment:
snwredist should probably not be set here

Reply:
I leave it to you to review and update the snw set_nml settings.  I updated the defaults, so all the snw flags were basically off.  So the set_nml.snw* files needed to be updated.

Reply:
I will change the option file appropriately.